MAINTAINERS.md

```diff
@@ -0,0 +1,37 @@
+# Podman Maintainers
+
+[GOVERNANCE.md](https://github.com/containers/podman/blob/main/GOVERNANCE.md)
+describes the Podman project's governance and the Project Roles used below.
+
+## Maintainers
+
+| Maintainer           | GitHub ID                                                | Project Roles     | Affiliation                               |
+| -------------------- | -------------------------------------------------------- | ----------------- | ----------------------------------------- |
+
+| Brent Baude          | [baude](https://github.com/baude)                        | Core Maintainer   | [Red Hat](https://www.github.com/redhat/) |
+| Ygal Blum            | [ygalblum](https://github.com/ygalblum)                  | Maintainer        | [Red Hat](https://www.github.com/redhat/) |
+| Jake Correnti        | [jakecorrenti](https://github.com/jakecorrenti)          | Reviewer          | [Red Hat](https://www.github.com/redhat/) |
+| Ashley Cui           | [ashley-cui](https://github.com/ashley-cui)              | Maintainer        | [Red Hat](https://www.github.com/redhat/) |
+| Nalin Dahyabhai      | [nalind](https://github.com/nalind)                      | Core Maintainer   | [Red Hat](https://www.github.com/redhat/) |
+| Jason Greene         | [n1hility](https://github.com/n1hility)                  | Reviewer          | [Red Hat](https://www.github.com/redhat/) |
```

Comment:
Jason has not been active in a while here I think? Does he want to be reviewer? If we assign roles with responsibilities then we should likely ask everyone here if they are good with that but maybe you have done that?

But I guess this question applies to everyone here really?